powerauth-push-client/src/main/java/io/getlime/push/client/PushServerClient.java

```diff
@@ -323,17 +323,13 @@ public ObjectResponse<CreateCampaignResponse> createCampaign(String appId, PushM
      * @throws PushServerClientException In case of network, response / JSON processing, or other IO error.
      */
     public boolean deleteCampaign(Long campaignId) throws PushServerClientException {
-        try {
-            String campaignIdSanitized = URLEncoder.encode(String.valueOf(campaignId), "UTF-8");
+        String campaignIdSanitized = URLEncoder.encode(String.valueOf(campaignId), StandardCharsets.UTF_8);
```

Comment:
Can we add `final` modifiers to the code we update, wherever applicable? Thanks.